web/src/interfaces.ts

```diff
@@ -1,4 +1,57 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 export interface Image {
```

Comment:
Add doc for this new interface (could be done separately as this is not added in this PR).

Reply:
@xlai20 Yep, will do in another PR! This one was merged previously before we decided to add docstrings to everything we export and I didn't want to add it to this PR haha